tools/src/main/java/io/github/ygojson/tools/dataprovider/impl/yugipedia/model/CardNumber.java

```diff
@@ -0,0 +1,69 @@
+package io.github.ygojson.tools.dataprovider.impl.yugipedia.model;
+
+import io.github.ygojson.model.data.definition.localization.Region;
+
+/**
+ * Represents the card number or set-prefix data.
+ * <br>
+ * Follows the rules of the <a href="https://yugipedia.com/wiki/Card_Number">Yugipedia CardNumber</a> description.
+
+ * @param stringValue the cardNumber as a String
+ * @param setCode the code or abbreviation of the set this prints belongs to
+ * @param regionCode the region code
+ * @param printNumberPrefix the prefix for the print number (i.e., if multiple subsets or promotional cards)
+ * @param printNumber the actual print number (without zeroes)
```

Comment:
This should be named `setPositionNumber` instead?

Reply:
Not really, it is more clear `printNumber`, as there are position 0 and some special editions follow up a cross-set numbering.